skrf/networkSet.py

```diff
@@ -1270,7 +1271,7 @@ def sel(self, indexers: Mapping[Any, Any] = None) -> NetworkSet:
                 ntwk_list.append(k)
 
         if ntwk_list:
-            return NetworkSet(ntwk_list)
+            return ntwk_list[0] if len(ntwk_list) == 1 else NetworkSet(ntwk_list)
```

Comment:
I understand, but that may break the code of people always expecting a NetworkSet...

Reply:
Yeah, I guess so. It seems like the right behavior to me, but it's definitely a breaking change for anyone depending on it to work as-is. What's your preference?

Reply:
Don't do this, as the user is forced to check all the time if the return value is a `Network` or a `NetworkSet`.